addons/website_slides/static/lib/pdfslidesviewer/PDFSlidesViewer.js

```diff
@@ -148,6 +148,23 @@ var PDFSlidesViewer = (function(){
     PDFSlidesViewer.prototype.toggleFullScreen = function(){
         var el = this.canvas;
 
+        var $PDFViewer = $("div#PDFSlideViewer")
+        var $navigation_footer = $('div#PDFViewer > div.navbar-fixed-bottom')
+
+        $(document).unbind('webkitfullscreenchange mozfullscreenchange fullscreenchange')
+            .bind('webkitfullscreenchange mozfullscreenchange fullscreenchange', function() {
+                var state = document.fullScreen || document.mozFullScreen || document.webkitIsFullScreen || document.msFullScreen;
+                if (!state) {
+                    $navigation_footer.removeAttr('style');
+                    $PDFViewer.unbind('click');
+                } else {
+                    $navigation_footer.css('z-index', 2147483647);
```

Comment:
Why this HUGE number as `z-index` ? I guess this is to display the footer on the fullscreen, but why this particular number ?